video/out/gpu/video.c

```diff
@@ -466,10 +466,6 @@ const struct m_sub_options gl_video_conf = {
         {"gpu-shader-cache-dir", OPT_STRING(shader_cache_dir), .flags = M_OPT_FILE},
         {"gpu-hwdec-interop",
             OPT_STRING_VALIDATE(hwdec_interop, ra_hwdec_validate_opt)},
-        {"gamut-warning", OPT_REMOVED("Replaced by --gamut-mapping-mode=warn")},
-        {"gamut-clipping", OPT_REMOVED("Replaced by --gamut-mapping-mode=desaturate")},
-        {"tone-mapping-desaturate", OPT_REMOVED("Replaced by --tone-mapping-mode")},
-        {"tone-mapping-desaturate-exponent", OPT_REMOVED("Replaced by --tone-mapping-mode")},
```

Comment:
I don't see how having to scour through `interface-changes.rst` is more helpful than this.

I'm also having a hard time seeing what the purpose of this even is. Just removing code for the sake of it?

Reply:
Please refer to https://github.com/mpv-player/mpv/pull/14382 for the discussion, this should give you idea, why this PR exists.